Dfe.Data.SearchPrototype/SearchForEstablishments/SearchResponseStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Dfe.Data.SearchPrototype.SearchForEstablishments;
+
+public enum SearchResponseStatus
+{
+    Success,
+    NoResults,
+    InvalidRequest,
+    SearchServiceError,
+    DataFormatError
+}
```

Comment:
This is the actual main thing I've added - this status and the setting of it by the use-case and the mapper